packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -604,7 +605,9 @@ module.exports = (
       timeToRead: {
         type: `Int`,
         resolve(markdownNode) {
-          return getHTML(markdownNode).then(timeToRead)
+          return getHTML(markdownNode).then(html =>
+            timeToRead(markdownNode, html, calcTimeToRead)
```

Comment:
it is more easy to understand if you rename the utils function
```suggestion
            timeToReadCalculator(markdownNode, html, timeToRead)
```

Reply:
That is a great point. I thought it was confusing too to be honest and didn't think about renaming the util function. Let me know if you like the new naming 👍